### ISSUE TITLE
"No episodes found" when running TV show related commands

### ISSUE THREAD

**Issue by reysic** [mateoradman/bazarr-bulk#33]
Running on Windows 11, bb v0.1.5, Bazarr version 1.4.3. I'm able to successfully run `movies` commands however `tv-shows` commands return "No episodes found" even though I have series in Bazarr.

![movies-cmd](https://github.com/mateoradman/bazarr-bulk/assets/36483043/2a23115a-bf82-4a7e-9d56-d9ec9a8ad21a)
![tv-shows-cmd](https://github.com/mateoradman/bazarr-bulk/assets/36483043/1d4d09d5-ac2d-4a04-a661-5d3da9e0cc72)

Let me know if there's any other info that would be helpful to provide.

Thanks!

**Comment by mateoradman** [comment]
Hi @reysic, many thanks for reporting this bug. I've looked at the code briefly and I think I found the root of this problem. It seems like the code returns from a function in case it encounters a tv show without episodes, instead of just continuing to the next iteration of the for-loop

```rust
            if num_episodes == 0 {
                sub_pb.finish_with_message("No episodes found");
                return Ok(());
            }
```

should be:

```rust
            if num_episodes == 0 {
                sub_pb.finish_with_message("No episodes found");
                continue;
            }
```

Based on the output you've provided, it seems like your very first tv show is without episodes. The interim solution would be to skip this tv show using `--offset 1` CLI flag. 

I will fix this problem and release a new version asap.
